src/main/java/com/easypost/model/CreditCardFund.java

```diff
@@ -0,0 +1,42 @@
+package com.easypost.model;
```

Comment:
Why is this an object? I don't believe this is an actual EasyPost object.

Reply:
This is the return response after the user funds a credit card. We need this class for GSON serialization.